internal/resolution/resolve.go

```diff
@@ -17,38 +17,38 @@ import (
 	"github.com/google/osv-scanner/pkg/models"
 )
 
-type ResolutionVuln struct {
-	Vulnerability models.Vulnerability
-	DevOnly       bool
+type Vulnerability struct {
+	OSV     models.Vulnerability
```

Comment:
I feel we can have a better name.. but I don't have a good suggestion neither...

Reply:
Yeah, it's not great. I settled on `OSV` because it is the format of the structure.